src/shared/components/ShareDropdown/ShareDropdown.tsx

```diff
@@ -0,0 +1,89 @@
+import { Dropdown, DropdownButton, DropdownContent } from '@meemoo/react-components';
+import { Button, IconName, Tabs } from '@viaa/avo2-components';
+import React, { FC, useState } from 'react';
+
+import { useTabs } from '../../hooks';
+import useTranslation from '../../hooks/useTranslation';
+import ShareWithColleagues from '../ShareWithColleagues/ShareWithColleagues';
+import {
+	ShareUserInfo,
+	ShareUserInfoRights,
+} from '../ShareWithColleagues/ShareWithColleagues.types';
+
+import './ShareDropdown.scss';
+
+type ShareDropdownProps = {
+	users: ShareUserInfo[];
+	onAddNewUser: (info: Partial<ShareUserInfo>) => void;
+	onEditRights: (info: ShareUserInfo, newRights: ShareUserInfoRights) => void;
+	onDeleteUser: (info: ShareUserInfo) => void;
+};
+const ShareDropdown: FC<ShareDropdownProps> = ({
+	users,
+	onAddNewUser,
+	onEditRights,
+	onDeleteUser,
+}) => {
+	const { tText } = useTranslation();
+	const [isShareDropdownOpen, setIsShareDropdownOpen] = useState<boolean>(false);
+	const [tab, setActiveTab, tabs] = useTabs(
+		[
+			{
+				id: 'colleagues',
+				label: tText('shared/components/share-dropdown/share-dropdown___collegas'),
+				icon: IconName.userTeacher,
+			},
+			{
+				id: 'pupils',
+				label: tText('shared/components/share-dropdown/share-dropdown___leerlingen'),
+				icon: IconName.userStudent,
+			},
+		],
+		'colleagues'
```

Comment:
maybe change this to 'pupils' so we don't have to come back to this in the other task